src/rest_settings.ts

```diff
@@ -66,6 +66,12 @@ const paths = {
   correlationManualTasks: `/correlations/${params.correlationId}/manual_tasks`,
   processModelCorrelationManualTasks: `/process_models/${params.processModelId}/correlations/${params.correlationId}/manual_tasks`,
   finishManualTask: `/processes/${params.processInstanceId}/correlations/${params.correlationId}/manual_tasks/${params.manualTaskInstanceId}/finish`,
+  // Task
+  allSuspendedTasks: '/suspended_tasks',
+  processModelTasks: `/process_models/${params.processModelId}/tasks`,
```

Comment:
```suggestion
  waitingProcessModelTasks: `/process_models/${params.processModelId}/tasks`,
```

Ich weis, das wurde in den anderen Routen auch nicht so benannt. Steht aber auf der TODO- Liste 😉